containers/ecr-viewer/src/app/utils/evaluate/fhir-paths.ts

```diff
@@ -389,14 +385,9 @@ const _fhirPathMappings: { [K in FhirPathKeys]: Omit<FhirPath<K>, "name"> } = {
     path: "entry.resource.Composition.section.where(code.coding.code = %code).entry",
   },
 
-  conditionCode: {
-    type: "CodeableConcept",
-    path: "Condition.code",
-  },
-
   facilityContact: {
     type: "string",
-    path: "entry.resource.Location.telecom.where(system = 'phone').value",
+    path: "entry.resource.Location.first().telecom.where(system = 'phone').value",
```

Comment:
should we add a ticket in the backlog for better dealing with facility info gathering? (this one seems especially dicey to me given there can be many locations on an eCR)